lib/oauth2/error.rb

```diff
@@ -16,7 +16,8 @@ def initialize(response)
         message << "#{@code}: #{@description}"
       end
 
-      message << response.body
+      #https://github.com/intridea/oauth2/issues/155
+      message << response.body.force_encoding("UTF-8")
```

Comment:
1.9.3-p484 :041 > f = File.open("/tmp/ascii-8bit", "r:ASCII-8BIT")
     => #<File:/tmp/ascii-8bit> 
    1.9.3-p484 :042 > d = f.read
     => "ASCII-8BIT\n\xC2\xA9 2011 AACR \xE2\x88\x9E\n" 
    1.9.3-p484 :043 > d.encoding
     => #<Encoding:ASCII-8BIT> 
    1.9.3-p484 :044 > aa << d
    Encoding::CompatibilityError: incompatible character encodings: UTF-8 and ASCII-8BIT
    	from (irb):44
    	from /Users/mavenlink/.rvm/rubies/ruby-1.9.3-p484/bin/irb:12:in `<main>'
    1.9.3-p484 :045 > aa << d
    Encoding::CompatibilityError: incompatible character encodings: UTF-8 and ASCII-8BIT
    	from (irb):45
    	from /Users/mavenlink/.rvm/rubies/ruby-1.9.3-p484/bin/irb:12:in `<main>'
    1.9.3-p484 :046 > aa << d.force_encoding("UTF-8")
     => "fo\xC2ooASCII-8BIT\n© 2011 AACR ∞\n" 
    1.9.3-p484 :047 > puts aa
    fo?ooASCII-8BIT
    © 2011 AACR ∞
     => nil 

I can try to get a spec on it i reckon?